files/scripts/update-kubernetes-system-services.sh

```diff
@@ -40,13 +40,13 @@ fi
 source "${GEN3_HOME}/gen3/gen3setup.sh"
 #gen3 gitops tfplan $@
 
-kube_proxy_version="1.14.7"
+kube_proxy_version="1.15.11"
 kube_proxy_image="602401143452.dkr.ecr.us-east-1.amazonaws.com/eks/kube-proxy:v${kube_proxy_version}"
 
-coredns_version="1.3.1"
+coredns_version="1.6.6"
 coredns_image="602401143452.dkr.ecr.us-east-1.amazonaws.com/eks/coredns:v${coredns_version}"
 
-cni_image="https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/release-1.5/config/v1.5/aws-k8s-cni.yaml"
+cni_image="https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/release-1.6/config/v1.6/aws-k8s-cni.yaml"
 
 
```

Comment:
I believe all EKS clusters come preloaded with CoreDNS/Kube-proxy/CNI, so we should get rid of one of the scripts, you may want to remove `update-kubernetes-system-services.sh` (this file) and leave the `kube-setup` one since it lines better with gen3